examples/README.md

```diff
@@ -15,6 +18,15 @@
   - `3_hl7_translation_final`
 - File extension: either `hl7` or `fhir`
 
+## Routing of these files
```

Comment:
I like this section, but consider _also_ referencing and linking to the ADR.

Reply:
Added